### DISCUSSION TITLE
[Weekly Review Report]: Nov 23 - 29

### DISCUSSION THREAD

**Discussion by mkutmon** [wikipathways/wikipathways-help#146] [curation-reports]
### Recent changes

 * All recent changes checked. All pathways are approved again. 
 * Would be good to add  the reference from the description also in the reference list [WP369](https://classic.wikipathways.org/index.php/Pathway:WP369)

### Under construction

 * @marvinm2: is this pathway ready to be approved? [WP5482](https://classic.wikipathways.org/index.php/Pathway:WP5482)
 * changed one tag from under construction to needs work [WP5486](https://classic.wikipathways.org/index.php/Pathway:WP5486)
 * tagged one tutorial pathway for deletion [WP5498](https://classic.wikipathways.org/index.php/Pathway:WP5498)

### New pathways

 * No new pathways except test

### Deleted Pathways

 * Deleted two pathways that were tagged a while ago - not changed since

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

### Additional notes

- featured pathways are not checked if the latest version is tagged anymore, I think. Only approved. Is this still needed?
- test search works but thumbnail image has not been created for that pathway
- for new users that don't have an approved pathway yet, do we also contact them by email then? (Classic site: Kel1821)

@susancoort is next up

**Comment by khanspers** [comment]
Thanks @mkutmon! For WP369, I added the reference from the description as a literature reference.

For featured pathways, we are no longer updating those on classic, that task has been removed from the QA protocol a while ago. 

For new users, we decided to contact them when their pathway or edit is approved, and we have a [template email](https://docs.google.com/document/d/1WJOqBVsU9dkJb9NNsvVXusaBVTAcx3HQngzrisMv1F8/edit?usp=sharing). But the announcement of new authors in the #new-authors Slack channel seems to have stopped.
